apps/web/src/app/(signing)/sign/[token]/sign-dialog.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
`sign-dialog.tsx` doesn't seem to be used in `form.tsx`. is that intentional?

Reply:
Nope, I fixed it